### ISSUE TITLE
Seq. Editor: Initial keyboard focus issue: Ctrl+A selects, but then Del fails to delete

### ISSUE THREAD

**Issue by lunakid** [lunakid/MuLab#7]
When entering the Sequence Editor (from the Composer, either by double-clicking a part or clicking EDIT), the keyboard focus appears to not be fully initialized or set on the Note Editor:

E.g. `Ctrl+A` does select all notes, as expected, but then pressing `Delete` does nothing.
To actually delete the selected notes now with `Delete`, you first have to click into the Note Editor explicitly. (Which then has the adverse effect of deselecting the notes, so they have to be selected again. Pressing `Delete` will then work, deleting them, as it should have the first time, too.)

**Comment by lunakid** [comment]
Closing, as per Jo's mail:

> M8.0.64 should fix 2 other issues you reported: The rewind issue and the initial keyboard focus in the seq editor.
